pkg/cmd/environment/list/list.go

```diff
@@ -28,14 +28,14 @@ func NewCmdList(f factory.Factory) *cobra.Command {
 				return err
 			}
 
-			envResources, err := client.Environments.Get(environments.EnvironmentsQuery{
-				Take: 1,
-			})
+			envResources, err := client.Environments.Get(environments.EnvironmentsQuery{})
 			if err != nil {
+				f.Spinner().Stop()
```

Comment:
Hmmm, this spinner is getting a bit awkward, I wonder if we can do something to help with defer etc. I have no solutions at the moment, but it's something to think about in the background